src/index.html

```diff
@@ -21,7 +24,394 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__budy">
+    <header class="header">
+      <a
+        href="#"
+        class="header_logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyoheader"
```

Comment:
The alt text 'Moyoheader' seems incorrect or unclear. It should describe the image content more accurately.